src/V3Param.cpp

```diff
@@ -227,6 +352,43 @@ class ParamVisitor : public AstNVisitor {
             }
         }
     }
+    void relinkPinsByName(AstPin* startpinp, AstNodeModule* modp) {
+        std::map<string, AstVar*> nameToPin;
+        for (AstNode* stmtp = modp->stmtsp(); stmtp; stmtp = stmtp->nextp()) {
+            if (AstVar* varp = VN_CAST(stmtp, Var)) {
+                if (varp->isIO() || varp->isGParam() || varp->isIfaceRef()) {
+                    nameToPin.insert(make_pair(varp->name(), varp));
+                }
+            }
+        }
+        for (AstPin* pinp = startpinp; pinp; pinp = VN_CAST(pinp->nextp(), Pin)) {
+            if (AstVar* varp = pinp->modVarp()) {
+                std::map<string, AstVar*>::const_iterator varIt = nameToPin.find(varp->name());
+                UASSERT_OBJ(varIt != nameToPin.end(), varp,
+                            "Not found in " << modp->prettyNameQ());
+                pinp->modVarp(varIt->second);
+            }
+        }
+    }
+    // Check if parameter setting during instantiation is simple enough for hierarchical verilation
+    void checkSupportedParam(AstNodeModule* modp, AstPin* pinp) const {
+        // InitArray and AstParamTypeDType are not supported because that can not be set via -G
+        // option.
+        if (pinp->modVarp()) {
+            bool supported = false;
+            if (AstConst* constp = VN_CAST(pinp->exprp(), Const)) {
+                supported = !constp->isOpaque();
+            }
+            if (!supported) {
+                pinp->v3error(modp->origName()
```

Comment:
```suggestion
                pinp->v3error(AstNode::prettyNameQ(modp->origName)
```

Ditto elsewhere - basically errors should always use a pretty and be quoted.

Reply:
3452508e25e5f140578f8f19e54bb7fc81dcad67 uses prettyNameQ()